dist/serializer/VectorSerializer.js

```diff
@@ -290,11 +297,11 @@ var VectorSerializer = exports.VectorSerializer = function (_BaseSerializer) {
         radius: olRegularShape.getRadius(),
         radius2: olRegularShape.getRadius2(),
         rotateWithView: olRegularShape.getRotateWithView(),
-        rotation: olRegularShape.getRotation(),
+        rotation: rotation,
         scale: olRegularShape.getScale(),
         snapToPixel: olRegularShape.getSnapToPixel(),
         stroke: _this.writeStrokeStyle(olRegularShape.getStroke()),
-        graphicName: getGraphicName()
+        graphicName: graphicName
       };
     };
```

Comment:
Not sure why this file came up in the PR, according to the `.gitignore` the `dist` directory should be ignored.

Reply:
Perhaps it works differently on the main branch, this is supposed to be merged into the 2.0 branch. According to the history the dist files were always committed, so I kept that change.